CloneVkontakte/CloneVkontakte/UIView/ChangedView.swift

```diff
@@ -0,0 +1,46 @@
+// ChangedView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// Добавление расширения в SB для UIView
+class ChangedView: UIView {
+    // MARK: - Private Properties
+
+    @IBInspectable private var sdSet: CGSize {
```

Comment:
```suggestion
    @IBInspectable private var sdSet: CGSize {
```
sdSet - дай норм  название с окончанием Size